solutions/usaco-738.mdx

```diff
@@ -2,64 +2,158 @@
 id: usaco-738
 source: USACO Silver 2017 US Open
 title: Paired Up
-author: Sofia Yang
+author: Sofia Yang, Ryan Chou
 
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_pairup_silver_open17.html)
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_pairup_silver_open17.html)
 
+In the sample input, we have the following cows:
+$8, 5, 5, 2$, and there's two ways to pair these up.
+<ol>
+<li>$(8, 5), (5, 2)$</li>
+<li>$(8, 2), (5, 5)$</li>
+</ol>
+
+The first case would take $13$ time units to finish milking all cows.
+
+The second case would take $10$ time units to finish milking all cows.
+
+This test case consists of 3 distinct values: $8, 5, 2$, let's call them $A, B, C$. These would follow the paradigm $A < B < C$.
+
+Therefore, $A+C < B + C$.
+
+We should always pair up the greatest value ($C$), with the smallest value ($A$), in order to achieve the most optimal milking time.
```

Comment:
```suggestion
We should always pair up the greatest value ($C$), with the smallest value ($A$), in order to achieve the most optimal milking time.
```
It would be nice if you included a proof + maybe more intuition for this. Otherwise, looks good!

Reply:
whoops you already did